subsys/bluetooth/shell/audio.c

```diff
@@ -44,7 +44,7 @@ static const struct bt_codec_qos_pref qos_pref = BT_CODEC_QOS_PREF(true, BT_GAP_
 
 #if defined(CONFIG_BT_AUDIO_UNICAST_CLIENT)
 static struct bt_audio_unicast_group *default_unicast_group;
-static struct bt_codec *rcodecs[2][CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT];
+static struct bt_codec rcodecs[2][20];
```

Comment:
Could '20' be changed to a define MAX_xxx?

Reply:
We can do that, but what improvement would that be? It is basically just an arbitrary value designed to support many different rcodecs for the shell. It probably should be smaller, as having 20 PAC records is quite insane. We could also consider just removing it, as the `rcodecs` is not used in the shell anyways. This might actually be true for most of the samples

Reply:
> We can do that, but what improvement would that be?

Less magic numbers to confuse :) But yes, maybe it should (also) be made lower

Reply:
That makes sense, however I decided to go with the option of deleting the storing of the codes, as they were all unused.